src/eth1spec/evm/instructions.py

```diff
@@ -62,12 +68,28 @@ def sstore(evm: Evm) -> None:
     OutOfGasError
         If `evm.gas_left` is less than `20000`.
     """
-    evm.gas_left = subtract_gas(evm.gas_left, U256(20000))
-
-    k = pop(evm.stack)
-    v = pop(evm.stack)
-
-    evm.env.state[evm.current].storage[k.to_be_bytes32()] = v
+    key = pop(evm.stack)
+    val = pop(evm.stack)
+    curr_storage = evm.env.state[evm.current].storage
```

Comment:
Is this necessary? Or can we just used the `vm.env.state[evm.current].storage.get(...)` form?